src/Core/Prefab/PlanarView.js

```diff
@@ -148,6 +148,12 @@ function PlanarView(viewerDiv, extent, options = {}) {
 
     this._renderState = RendererConstant.FINAL;
     this._fullSizeDepthBuffer = null;
+    this.addFrameRequester(MAIN_LOOP_EVENTS.BEFORE_RENDER, () => {
```

Comment:
Can you elaborate why you need that?

Reply:
Because `_fullSizeDepthBuffer` will never get invalidated otherwise.

So we clear it before each render, and `_fullSizeDepthBuffer` only act as a cache (ie: if you request depth values several times without redrawing the scene the depth buffer will only be read once).